packages/nouns-contracts/contracts/governance/NounsDAOV3Proposals.sol

```diff
@@ -225,23 +225,27 @@ library NounsDAOV3Proposals {
         checkProposalTxs(txs);
         uint256 proposalId = ds.proposalCount = ds.proposalCount + 1;
 
-        (uint256 votes, address[] memory signers) = verifySignersCanBackThisProposalAndCountTheirVotes(
-            ds,
-            proposerSignatures,
-            txs,
-            description,
-            proposalId
-        );
-
         uint256 adjustedTotalSupply = ds.adjustedTotalSupply();
-        uint256 propThreshold = checkPropThreshold(ds, votes, adjustedTotalSupply);
+
+        uint256 propThreshold = proposalThreshold(ds, adjustedTotalSupply);
+
         NounsDAOStorageV3.Proposal storage newProposal = createNewProposal(
             ds,
             proposalId,
             propThreshold,
             adjustedTotalSupply,
             txs
         );
+
+        (uint256 votes, address[] memory signers) = verifySignersCanBackThisProposalAndCountTheirVotes(
```

Comment:
wonder if it's worth adding an inline comment above this emphasizing this must happen after proposal creation?